src/ResourceManager/Sql/Commands.Sql/Elastic Pools/Model/AzureSqlElasticPoolModel.cs

```diff
@@ -78,6 +84,28 @@ public class AzureSqlElasticPoolModel
         /// </summary>
         public int? DatabaseDtuMin { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Sku capacity of the elastic pool
+        /// </summary>
+        public int? Capacity { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Minumum capacity of the elastic pool
+        ///    For Dtu pool, it is the min Database Dtu of the pool; for Vcore pool, it is the min vcore capacity of the pool
+        /// </summary>
+        public double? MinCapacity { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Maximum capacity of the elastic pool
+        ///    For Dtu pool, it is the max Database Dtu of the pool; for Vcore pool, it is the max vcore capacity of the pool
+        /// </summary>
+        public double? MaxCapacity { get; set; }
+
+        /// <summary>
+        /// Gets or sets the amount of storage in bytes the elastic pool has in bytes
+        /// </summary>
+        public long? MaxSizeBytes { get; set; }
```

Comment:
Seems redundant since there is already StorageMB

Reply:
@jaredmoo If remove the MaxSizeBytes for elasticPool model, and apply storage value to StorageMB when construct ElasticPoolModel.  In the PUT pool request body, it only include MaxSizeBytes in the properties, and MaxSizeBytes will be automatically set to a negative number "-2147483648", the request would fail.